### DISCUSSION TITLE
Align right parenthesis on a different line to front

### DISCUSSION THREAD

**Discussion by linw789** [LunarWatcher/auto-pairs#38] [Q&A] [answered]
input: 
```
foo( | ) (press <CR> at |)
```

output: 
```
foo(
   |
   )
```

Is there way to configure so the right parenthesis goes to alight with foo? Like this:
```
foo(
    |
)
```

**Comment by LunarWatcher** [comment, marked as answer]
If it's a cino-compatible language, `set cino+=m1` should work. That type if alignment is pretty standard for C languages by default though, so I'm assuming it's one of them. (Vim doesn't remotely try to align in Python by default).

If `set cino+=m1` doesn't work, what language are we talking about? A plugin may be needed to get an indentexpr that supports that.

**Reply by linw789** [reply]
`set cino+=m1` worked. I'm working with Rust right now.

BTW, thanks for forking auto-pairs and continuing maintaining it.
